util/config.py

```diff
@@ -109,6 +109,8 @@ def __repr__(self):
                 enable_data_package_reference=False,
                 enable_tokens=False,
                 enable_tape_archive=False,
+                inactivity_cutoff_months=3,
+                inactivity_notification=0,
```

Comment:
Can you change the format of this configuration item so that it uses the same format as the other feature switch parameters, for consistency? Something like: enable_inactivity_notifications=False